docs/content/setup/accounts-and-security.md

```diff
@@ -7,15 +7,15 @@ pre: "<i class='fas fa-user-lock'></i> "
 tags: ["usage", "password"]
 ---
 
-## Security in Infection Monkey
+## Security in the Infection Monkey
 
-The first time you launch Monkey Island (Infection Monkey CC server), you'll be prompted to create an account and secure your island. After your account is created, the server will only be accessible via the credentials you chose.
+The first time you launch Monkey Island (the Infection Monkey CC server), you'll be prompted to create an account and secure your island. After account creation, the server will only be accessible via the credentials you entered.
 
-If you want island to be accessible without credentials press *I want anyone to access the island*. Please note that this option is insecure: you should only pick this for use in development environments.
+If you want an island to be accessible without credentials, press *I want anyone to access the island*. Please note that this option is insecure, and you should only use it in development environments.
 
-## Resetting account credentials
+## Resetting your account credentials
 
-To reset credentials edit `monkey_island\cc\server_config.json` by deleting `user` and `password_hash` variables. Then restart the Monkey Island server and you should be prompted with registration form again.
+To reset your credentials, edit `monkey_island\cc\server_config.json` by deleting the `user` and `password_hash` variables. Then restart the Monkey Island server, and you should be prompted with a registration form again.
```

Comment:
What about "When you restart the Monkey Island server, you will be prompted with the registration form."